app/controllers/projects_controller.rb

```diff
@@ -56,14 +56,39 @@ def category_projects
   # GET /user/user_id/projects?admin=1\0
   def user_projects
 	  if params[:admin].present?
-		  @pagy, @projects = pagy(Project.of_user_by_role(params[:user_id],params[:admin]).order(created_at: :desc))
+		  @pagy, @projects = pagy(Project.of_user_by_role(params[:user_id],params[:admin]).includes(:project_status).order(created_at: :desc))
 	  else
-		  @pagy, @projects = pagy(Project.of_user(params[:user_id]).order(created_at: :desc))
-	  end
+		  @pagy, @projects = pagy(Project.of_user(params[:user_id]).includes(:project_status).order(created_at: :desc))
+    end
+
+    @serializer_options[:include] = [:project_status]
+    @serializer_options.merge!(pagination_options(@pagy))
```

Comment:
effettua l'associazione e renderizza in jason i project_status all'interno degli includes, ma non riporta l'attributo name della tabella project_status.
provare con postman per credere.